runtime/klaos/src/lib.rs

```diff
@@ -764,6 +764,7 @@ construct_runtime!(
 	}
 );
 
+#[allow(dead_code)] // https://github.com/moonbeam-foundation/moonbeam/blob/26a88a553563647992f39fbd1cce3d45a363e991/runtime/moonbeam/src/lib.rs#L1585-L1614
```

Comment:
shall we add these links to the issue on the github project for future reference?